resources/sdk/pureclouddotnet-guest/templates/README.mustache

```diff
@@ -3,6 +3,8 @@ title: Platform API Guest Chat Client SDK - .NET
 ---
 [![NuGet Badge](https://buildstats.info/nuget/PureCloudPlatform.Client.V2.Guest)](https://www.nuget.org/packages/PureCloudPlatform.Client.V2.Guest/)
 
+[Platform release notes](releaseNotes.md)
```

Comment:
I'd call this something like "SDK release notes for v{{packageVersion}}". Including the version is a maybe, but I wouldn't say platform here because these release notes are meant for SDK consumers, not general Genesys Cloud platform release notes.

Reply:
Remember we were not going to include version in here to keep from confusing pepople